maint/ucx-embed.patch

```diff
@@ -0,0 +1,106 @@
+From 1e6ef317f2676d19627a40fe2cd09b1e62934538 Mon Sep 17 00:00:00 2001
+From: Ken Raffenetti <raffenet@mcs.anl.gov>
+Date: Wed, 18 Oct 2017 14:39:34 -0500
+Subject: [PATCH] config: Add embedded mode
+
+Add a configure flag to support building UCX as a subproject of
+another project. When configured for embedded mode, we avoid
+installing any header files.
+---
+ configure.ac        | 5 +++++
+ src/ucm/Makefile.am | 2 ++
+ src/ucp/Makefile.am | 2 ++
+ src/ucs/Makefile.am | 2 ++
+ src/uct/Makefile.am | 2 ++
+ 5 files changed, 13 insertions(+)
+
+diff --git a/configure.ac b/configure.ac
+index 74284add..1a257d64 100644
+--- a/configure.ac
++++ b/configure.ac
+@@ -77,6 +77,11 @@ AC_C_RESTRICT
+ m4_include([config/m4/pkg.m4])
+ PKG_PROG_PKG_CONFIG
+ 
```

Comment:
I see weird white spaces on my laptop on the above line and multiple places below.

Reply:
That whitespace is in the `configure.ac` file for UCX. It won't apply if I remove it.

Reply:
Sorry, I meant the `Makefile.am` files.

Reply:
Hmm, maybe I'm wrong. I think it's just the patchfile format looking weird. The patch itself applies clean and doesn't have any extra whitespace.